src/cmds/scripts/pbs_habitat.in

```diff
@@ -721,9 +722,15 @@ if [ -z "$PBS_HOME" ]; then
 	exit 1
 fi
 if [ ! -d "$PBS_HOME" ]; then
-	echo "*** $PBS_HOME directory does not exist"
-	echo "*** Run pbs_postinstall to create it"
-	exit 1
+	echo "*** WARNING: PBS_HOME not found in $PBS_HOME"
+	if [ -x "$PBS_EXEC/sbin/pbs_server" ]; then
+		component="server"
+	elif [ -x "$PBS_EXEC/sbin/pbs_mom" ]; then
+		component="execution"
+	else
+		echo "*** PBS will not create home in client installation"
```

Comment:
PBS_HOME is not required in a client installation. This block will be executed only in user error scenario. It would help the admin to debug the error.